OSDOCS-4265: Updates some ROSA cli items for CloudFront (#53514)
openshift/openshift-docs: snippets/oidc-cloudfront.adoc

//This snippet appears in the following assemblies:
//
// * ../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc

:_content-type: SNIPPET
[NOTE]
====

> If you want to obscure the OIDC endpoint for your cluster using CloudFront, you must use `rosa` CLI version 1.2.7 or later.

Reviewer: change to:
> As of ROSA CLI 1.2.8 and December 8, 2022, all new clusters have a new format for the OIDC provider endpoint URL. ROSA cluster OIDC provider URLs are no longer regional. AWS CloudFront is part of this implementation to improve access speed, reduce latency and improve resiliency.

> This feature can only be used when creating a cluster. There are no migration options for existing clusters.

Reviewer: change to:
> This change is only available to new clusters as for ROSA CLI 1.2.8 and December 8, 2022. There are no supported migration paths for existing OIDC-provider configurations.

> As of ROSA CLI 1.2.7, all new clusters have a new format for the OIDC provider endpoint URL. {product-title} cluster OIDC provider URLs are no longer regional. The AWS CloudFront implementation provides improved access speed and resiliency and reduces latency.

Reviewer 2: Up to you on this, trying to make it more direct and less passive feeling

```suggestion
ROSA CLI 1.2.7 introduces changes to the the OIDC provider endpoint URL format for new clusters. {product-title} cluster OIDC provider URLs are no longer regional. The AWS CloudFront implementation provides improved access speed and resiliency and reduces latency.
```

> As of ROSA CLI 1.2.8, all new clusters have a new format for the OIDC provider endpoint URL. {product-title} cluster OIDC provider URLs are no longer regional. The AWS CloudFront implementation provides improved access speed and resiliency as well as reduces latency.

Reviewer 3: If I'm not wrong we have the new format for OIDC provider endpoint URL from **v1.2.7**.
`oadler@dhcp-4-86:rosa ((7fc46054...))$ ./rosa version
1.2.7
oadler@dhcp-4-86:rosa ((7fc46054...))$ ./rosa create cluster ...
...
OIDC Endpoint URL:          https://dvbwgdztaeq9o.cloudfront.net/20f03tas19t16ktu1us0re9pcoehr11c`
